### ISSUE TITLE
The output of my real data is wrong

### ISSUE THREAD

**Issue by Fateme2020** [SoftWiser-group/iTag#8]
After following previous issues answers I ran iTag in proper device but all results are zero for my data. here is my output is below . Can you help me whats wrong here? thanks you.
[0]
(1, 30, 17.905511811023622)
(1, 1, 1.0)
('unique_words:', 1489)
('common:', 0)
('unique_tags:', 3)
('index: ', 1492)
('tags index:', 1492)
('shared_index:', 1495)
('tag from :', 1492)
('sfs end:', 1495)
('sfs start:', 1496)
Train on 308 samples, validate on 35 samples
Epoch 1/60
 - 25s - loss: 12.7599 - dense_4_loss: 7.1258 - dense_5_loss: 5.6341 - dense_4_compute_precision: 0.0000e+00 - dense_4_compute_recall: 0.0000e+00 - dense_5_compute_precision: 0.0000e+00 - dense_5_compute_recall: 0.0000e+00 - val_loss: 11.4308 - val_dense_4_loss: 5.7979 - val_dense_5_loss: 5.6328 - val_dense_4_compute_precision: 0.0000e+00 - val_dense_4_compute_recall: 0.0000e+00 - val_dense_5_compute_precision: 0.0000e+00 - val_dense_5_compute_recall: 0.0000e+00
/usr/local/lib/python2.7/dist-packages/keras/engine/topology.py:2368: UserWarning: Layer gru_10 was passed non-serializable keyword arguments: {'initial_state': [<tf.Tensor 'gru_7/while/Exit_2:0' shape=(?, 150) dtype=float32>]}. They will not be included in the serialized model (and thus will be missing at deserialization time).
  str(node.arguments) + '. They will not be included '
/usr/local/lib/python2.7/dist-packages/keras/engine/topology.py:2368: UserWarning: Layer gru_11 was passed non-serializable keyword arguments: {'initial_state': [<tf.Tensor 'gru_8/while/Exit_2:0' shape=(?, 150) dtype=float32>]}. They will not be included in the serialized model (and thus will be missing at deserialization time).
  str(node.arguments) + '. They will not be included '
/usr/local/lib/python2.7/dist-packages/keras/engine/topology.py:2368: UserWarning: Layer gru_12 was passed non-serializable keyword arguments: {'initial_state': [<tf.Tensor 'gru_9/while/Exit_2:0' shape=(?, 150) dtype=float32>]}. They will not be included in the serialized model (and thus will be missing at deserialization time).
  str(node.arguments) + '. They will not be included '
Epoch 2/60
 - 19s - loss: 7.9630 - dense_4_loss: 2.3309 - dense_5_loss: 5.6320 - dense_4_compute_precision: 0.0394 - dense_4_compute_recall: 0.2435 - dense_5_compute_precision: 0.0000e+00 - dense_5_compute_recall: 0.0000e+00 - val_loss: 6.9859 - val_dense_4_loss: 1.3553 - val_dense_5_loss: 5.6306 - val_dense_4_compute_precision: 0.0000e+00 - val_dense_4_compute_recall: 0.0000e+00 - val_dense_5_compute_precision: 0.0000e+00 - val_dense_5_compute_recall: 0.0000e+00
Epoch 3/60
 - 19s - loss: 6.7999 - dense_4_loss: 1.1705 - dense_5_loss: 5.6294 - dense_4_compute_precision: 0.0344 - dense_4_compute_recall: 0.1851 - dense_5_compute_precision: 0.0000e+00 - dense_5_compute_recall: 0.0000e+00 - val_loss: 6.8060 - val_dense_4_loss: 1.1779 - val_dense_5_loss: 5.6280 - val_dense_4_compute_precision: 0.0420 - val_dense_4_compute_recall: 0.4000 - val_dense_5_compute_precision: 0.0000e+00 - val_dense_5_compute_recall: 0.0000e+00
Epoch 4/60
 - 19s - loss: 6.7105 - dense_4_loss: 1.0837 - dense_5_loss: 5.6268 - dense_4_compute_precision: 0.0518 - dense_4_compute_recall: 0.2597 - dense_5_compute_precision: 0.0000e+00 - dense_5_compute_recall: 0.0000e+00 - val_loss: 6.7228 - val_dense_4_loss: 1.0972 - val_dense_5_loss: 5.6256 - val_dense_4_compute_precision: 0.0000e+00 - val_dense_4_compute_recall: 0.0000e+00 - val_dense_5_compute_precision: 0.0000e+00 - val_dense_5_compute_recall: 0.0000e+00
Epoch 5/60
 - 19s - loss: 6.7061 - dense_4_loss: 1.0816 - dense_5_loss: 5.6245 - dense_4_compute_precision: 0.0457 - dense_4_compute_recall: 0.2370 - dense_5_compute_precision: 0.0000e+00 - dense_5_compute_recall: 0.0000e+00 - val_loss: 6.7421 - val_dense_4_loss: 1.1187 - val_dense_5_loss: 5.6234 - val_dense_4_compute_precision: 0.0436 - val_dense_4_compute_recall: 0.4286 - val_dense_5_compute_precision: 0.0000e+00 - val_dense_5_compute_recall: 0.0000e+00
Epoch 6/60
 - 19s - loss: 6.7027 - dense_4_loss: 1.0804 - dense_5_loss: 5.6223 - dense_4_compute_precision: 0.0482 - dense_4_compute_recall: 0.3442 - dense_5_compute_precision: 0.0000e+00 - dense_5_compute_recall: 0.0000e+00 - val_loss: 6.7151 - val_dense_4_loss: 1.0937 - val_dense_5_loss: 5.6214 - val_dense_4_compute_precision: 0.0000e+00 - val_dense_4_compute_recall: 0.0000e+00 - val_dense_5_compute_precision: 0.0000e+00 - val_dense_5_compute_recall: 0.0000e+00
Epoch 7/60
 - 19s - loss: 6.6876 - dense_4_loss: 1.0672 - dense_5_loss: 5.6204 - dense_4_compute_precision: 0.0529 - dense_4_compute_recall: 0.1916 - dense_5_compute_precision: 0.0000e+00 - dense_5_compute_recall: 0.0000e+00 - val_loss: 6.7209 - val_dense_4_loss: 1.1014 - val_dense_5_loss: 5.6195 - val_dense_4_compute_precision: 0.0434 - val_dense_4_compute_recall: 0.4000 - val_dense_5_compute_precision: 0.0000e+00 - val_dense_5_compute_recall: 0.0000e+00
Epoch 8/60
 - 19s - loss: 6.7129 - dense_4_loss: 1.0943 - dense_5_loss: 5.6186 - dense_4_compute_precision: 0.0487 - dense_4_compute_recall: 0.1786 - dense_5_compute_precision: 0.0000e+00 - dense_5_compute_recall: 0.0000e+00 - val_loss: 6.7198 - val_dense_4_loss: 1.1020 - val_dense_5_loss: 5.6178 - val_dense_4_compute_precision: 0.0857 - val_dense_4_compute_recall: 0.0286 - val_dense_5_compute_precision: 0.0000e+00 - val_dense_5_compute_recall: 0.0000e+00
full hit: 0.000000, precision@5: 0.000000, recall@5: 0.000000, f1@5: 0.000000

**Comment by stxupengyu** [comment]
Do you know where the dataset can be found? I'm a novice in text processing. I've been searching on Google for a long time and still haven't found it.
